pkg/ddc/alluxio/cache.go

```diff
@@ -16,19 +16,61 @@ limitations under the License.
 package alluxio
 
 import (
+	"fmt"
 	units "github.com/docker/go-units"
 	"github.com/fluid-cloudnative/fluid/pkg/ddc/alluxio/operations"
+	"github.com/fluid-cloudnative/fluid/pkg/utils"
 	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
+	"strings"
 )
 
 // queryCacheStatus checks the cache status
 func (e *AlluxioEngine) queryCacheStatus() (states cacheStates, err error) {
-	cacheCapacity, err := e.getCurrentCachedCapacity()
+	//cacheCapacity, err := e.getCurrentCachedCapacity()
+	//if err != nil {
+	//	e.Log.Error(err, "Failed to sync the cache")
+	//	return states, err
+	//}
+	dataset, err := utils.GetDataset(e.Client, e.name, e.namespace)
 	if err != nil {
-		e.Log.Error(err, "Failed to sync the cache")
+		e.Log.Error(err, "Failed to get dataset when query cache status")
 		return states, err
 	}
 
+	var (
+		totalCacheCapacity string = ""
+		usedCacheCapacity  string = ""
+		cachedPercentage   string = ""
+	)
```

Comment:
how about adding default value ?
totalCacheCapacity ”0.00kiB“
usedCacheCapacity "0.00kiB"
cachedPercentage "0.0%"

Reply:
I'm not sure setting `cachedPercentage` default to “0.0%” is a good idea. Take the following as an example:
Some user ran some app consuming the dataset before ufs initialization has done , and then the `usedCacheCapacity` is not 0, but `cachedPercentage` is still "0.0%"(default value) because `ufsTotal` is still empty. This may be a little confusing.
What do you think?

Reply:
How about tell user when initialization have not done.
```
if dataset.Status.usedCacheCapacity !=  $default
{
    if dataset.Status.UfsTotal != $default {
        usedInBytes, _ := units.FromHumanSize(usedCacheCapacity)
	ufsTotalInBytes, _ := units.RAMInBytes(dataset.Status.UfsTotal)
	cachedPercentage = fmt.Sprintf("%.1f%%", float64(usedInBytes)/float64(ufsTotalInBytes)*100.0)
    } else {
        usedInBytes, _ := units.FromHumanSize(usedCacheCapacity)
	cachedPercentage = fmt.Sprintf("%s", "Initializating")
    }
}
```

Reply:
Thanks for the suggestion. This looks good. Will update it